src/olympia/abuse/tests/test_cinder.py

```diff
@@ -1567,6 +1569,56 @@ def test_workflow_recreate_waffle_switch_off(self):
         assert ActivityLog.objects.count() == 0
 
 
+class TestCinderAddonHandledByLegal(TestCinderAddon):
+    CinderClass = CinderAddonHandledByLegal
+    # For rendering the payload to Cinder like CinderAddon:
+    # - 1 Fetch Addon authors
+    # - 2 Fetch Promoted Addon
+    expected_queries_for_report = 2
+    expected_queue_suffix = None
+
+    def test_queue(self):
+        extension = self._create_dummy_target()
+        assert self.CinderClass(extension).queue == 'legal-escalations'
```

Comment:
Nit: Is this string defined in a constant somewhere? Would make future updates a lot easier if we referenced the constant instead of the raw string.

Reply:
Probably. But that could be a separate issue.